flink-runtime/pom.xml

```diff
@@ -301,7 +301,7 @@ under the License.
 
 		<dependency>
 			<groupId>org.mockito</groupId>
-			<artifactId>mockito-inline</artifactId>
+			<artifactId>mockito-subclass</artifactId>
```

Comment:
mockito-inline stopped being released after 5.2.0, classes required by Flink tests were moved to mockito-subclass